<launch>
  	<node pkg="robot_localization" type="navsat_transform_node" name="navsat_transform_node" clear_params="true">

  		<!--Params currently set to default-->
		<param name="delay" value=3.0>
		<param name="magnetic_declination_radians" value=0>
		<param name="yaw_offset" value=0.0>
		<param name="broadcast_utm_transform" value=false>
		<param name="broadcast_utm_transform_as_parent_frame" value=false>
		<param name="use_odometry_yaw" value=false>
		<param name="wait_for_datum" value=false>
		<param name="datum" value=[0.0, 0.0, 0.0]><!--[latitude,longitude,heading(0.0=east)]-->

		<!--publishes to gps/filtered-->
		<param name="publish_filtered_gps" value=true>

		<!--Dont know where imu data is pusblished-->
		<remap from="imu/data" to="imu/data"/>
		<!--odometry/filtered is output from ekf-->
		<remap from="odometry/filtered" to="">
		<!--Assuming /odometry_navsatfix is topic of raw_gps converted to navsatfix msg__>
		<remap from="gps/fix" to="/odometry_navsatfix"/>

	</node>
</launch>

